package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@exabyte-io/esse.js",
-    "version": "2.3.0",
+    "version": "2020.10.19.2",
```

Comment:
Should this be changed? I didn't edit this file, i think the change might be from a different branch that got carried over from an older pull.

Reply:
should be 2020.10.19.2

Reply:
We should absolutely never be manually editing package-lock.json.  It's generated by node build tooling.

Reply:
In this case this is fine, i think.  There's no integrity checks in package-lock for the topmost package, so this should install without error.  We should figure out the rightway to regenerate the package-lock.json though, and document it in the readme.
It looks like it might be `npm install --package-lock-only` to rebuild the package-lock.json from the package.json file, but it needs to be tested.
https://docs.npmjs.com/cli/v6/commands/npm-install